components/Tests/CandidateTestView.tsx

```diff
@@ -0,0 +1,35 @@
+import {CandidateTest} from "../ApiFetch/TestFetcher";
+import scss from "../../pageStyles/testpage.module.scss";
+import TextEditorContainer from "../TextEditor/TextEditorContainer";
+import limitedAutoComplete from "../TextEditor/Settings/LimitedAutoComplete";
+import React from "react";
+import {getEditorProps} from "./TestEditorBuilder";
+import {AddTwoIntegersTest} from "./AddTwoIntegers";
+
+const EditorProps = getEditorProps(AddTwoIntegersTest);
+type CandidateTestProps = { props: CandidateTest };
+
+const CandidateTestView = ({props}: CandidateTestProps): JSX.Element => (
+    <div>
+        <h1 className={scss.testTitle}>Test {props.number}: {props.title}</h1>
+        <section className={scss.testPage}>
```

Comment:
we _probably_ don't need this outer `section` - the 2 inner sections should be enough

Reply:
This was so I could style both sections at the same time and keep them in the same container.